cross/rhash/Makefile

```diff
@@ -0,0 +1,23 @@
+PKG_NAME = RHash
+PKG_VERS = 1.3.5
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/rhash/RHash/archive
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
```

Comment:
please use `PKG_DIST_FILE` for a more clear filename than `v1.3.5.tar.gz` something like:
`PKG_DIST_FILE=$(PKG_NAME)-v$(PKG_VERS).$(PKG_EXT)`

see https://github.com/SynoCommunity/spksrc/blob/master/cross/rtmpdump/Makefile#L5 for an example